internal/spdxlicense/license_test.go

```diff
@@ -20,8 +20,8 @@ func TestIDParse(t *testing.T) {
 			"GPL-2.0",
 		},
 		{
-			"GPL-2+",
-			"GPL-2.0+",
+			"GPL-1+",
```

Comment:
With GPL-2+ replaced this test started to fail, therefore this change

Reply:
Can you elaborate more? What makes this change the _right_ change, beyond the fact that the tests pass now?

Reply:
With this PR `GPL-2.0+` is overwritten by `GPL-2.0-or-later`, the test above is meant to validate that the code handles `GPL-1+` -> `GPL-1.0+`, where the code should imply the zero digit, which might be omitted in a package.

The case for `GPL-2.0+` -> `GPL-2.0-or-later` got its own test at https://github.com/jonasagx/syft/blob/deprecate-gpl-2%2B-license/internal/spdxlicense/generate/generate_license_list_test.go#L149

Reply:
(That link to `generate_license_list_test.go` doesn't test `GPL-2.0+` -> `GPL-2.0-or-later`.)

>where the code should imply the zero digit

Isn't this tested by the prior test case as well? (it looks like this is exercised in both the 2nd and 3rd test case, if I'm following correctly)

Reply:
You're right. Sorry about the broken link above. I've updated the test to [[1](https://github.com/anchore/syft/pull/1009/files#diff-ef65afb470a7847837074d4821ae6f10bdd7afcb46641ad73cdaf132b418e1beL20-R25)].
```diff
		{
 			"GPL-2",
 			"GPL-2.0-only",
 		},
 		{
 			"GPL-2+",
 			"GPL-2.0-or-later",
 		},
```